<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaAdicionar</class>
 <widget class="QWidget" name="TelaAdicionar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h4&gt;Foto do Aluno&lt;/h4&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdicionarFoto">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar Foto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCadastrar">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>410</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>411</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;b&gt;Nome Completo do Aluno:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leNomeAluno"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;b&gt;Data de Nascimento:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateNascimento">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;b&gt;Sexo (masculino ou feminino)&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbMasculino">
        <property name="text">
         <string>Masculino</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbFeminino">
        <property name="text">
         <string>Feminino</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;b&gt;RG:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leRG">
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>XX.XXX.XXX-X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>&lt;b&gt;CPF:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leCPF">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>XXX.XXX.XXX-XX</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>&lt;b&gt;Nome do Responsável:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leNomeResponsavel"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>&lt;b&gt;Telefone para contato do responsável:&lt;/b&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leContato">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>(XX)XXXXX-XXXX</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
